volatility3/cli/__init__.py

```diff
@@ -827,7 +827,11 @@ def populate_requirements_argparse(
                 requirement,
                 volatility3.framework.configuration.requirements.ListRequirement,
             ):
-                additional["type"] = requirement.element_type
+                # Allow a list of integers, specified with convenient 0x hexadecimal format
```

Comment:
I'd kind of prefer fixes like this that affect everything to be separate pull requests if that's ok?  Just makes it easier to track down changes and all the rest of the code to stand or fall on its own.  This bit would get committed first, but essentially it affects more than just the mac process memory dump stuff.

Reply:
Sure, I will open a specific PR !